### ISSUE TITLE
Is it possible to use `[target.static]` without TS?

### ISSUE THREAD

**Issue by omohokcoj** [github/catalyst#300]
I'm following the v2 documentation (https://catalyst.rocks/guide-v2/targets) but the following code snippet doesn't work with JS:

```js
import { actionable, targetable, target } from '@github/catalyst'

export default actionable(targetable(class extends HTMLElement {
  [target.static] = ['fullAddressField'] // on html template there is a data-target="controller-name.fullAddressField" attr
  
  toggleAddressField () {
    this.fullAddressField.required = false
  }
}))
```

Is this possible to use this feature in JS or it works only via TS?

**Comment by keithamus** [comment]
It should be possible. If it doesn't work it may be a bug. I will add a test shortly.

**Comment by keithamus** [comment]
Thanks for replying back @omohokcoj! Glad you got this solved :raised_hands:

**Comment by omohokcoj** [comment]
@keithamus btw looks like the issue was not with my babel setup but just in general for JS in browser without any transpiler `static [target.static] = ['name']` is the correct way to define the targets (`static` keyword is missing in the documentation).


```html
<!doctype html>
<html>
<head>
  <meta charset="utf-8">
  <title>Untitled</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
</head>
<body>
  <!--
    JSPM Generator Import Map
    Edit URL: https://generator.jspm.io/#U2NhYGBkDM0rySzJSU1hcEjPLMkoTdJPTixJzKksLnEw0jPQM9BNSi1JBAC1ga7nKgA
  -->
  <script type="importmap">
  {
    "imports": {
      "@github/catalyst": "https://ga.jspm.io/npm:@github/catalyst@2.0.0-beta/lib/index.js"
    }
  }
  </script>
  <!-- ES Module Shims: Import maps polyfill for modules browsers without import maps support (all except Chrome 89+) -->
  <script async src="https://ga.jspm.io/npm:es-module-shims@1.5.1/dist/es-module-shims.js" crossorigin="anonymous"></script>
  <script type="module">
    import { controller, target } from "@github/catalyst";

    controller(class TestElem extends HTMLElement {
      static [target.static] = ['test'] // static keyword is required

      connectedCallback () {
        console.log(this.test)
      }
    })
  </script>
  <test-elem>
    <div data-target="test-elem.test">
    </div>
  </test-elem>
</body>
</html>
``` 

Anyway, thanks for maintaining such a useful library - github/catalyst with native web components looks much better for rails app than rails/stimulus 😄

**Comment by keithamus** [comment]
Thanks! I've fixed the docs in 173a24112109f7c4a9a7bf6b34078522952a2099
